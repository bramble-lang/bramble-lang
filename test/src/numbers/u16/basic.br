fn my_main() -> i64 {
    let x:u16 := 4u16;
    let y:u16 := test();
    let z:u16 := test2(16000u16);
    let z2:u16 := test2(32767u16);
    let z3:u16 := test2(65535u16);

    project::std::io::writeu16ln(x);
    project::std::io::writeu16ln(y);
    project::std::io::writeu16ln(z);
    project::std::io::writeu16ln(z2);
    project::std::io::writeu16ln(z3);
    
    return 0;
}

fn test() -> u16 {
    return 5u16;
}

fn test2(i: u16) -> u16 {
    return i;
}