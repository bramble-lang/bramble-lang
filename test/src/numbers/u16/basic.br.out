4
5
16000
32767
65535
